x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/api/ml_models/delete_ml_inference_pipeline.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { createApiLogic } from '../../../shared/api_logic/create_api_logic';
+import { HttpLogic } from '../../../shared/http';
+
+export interface DeleteMlInferencePipelineApiLogicArgs {
+  indexName: string;
+  pipelineName: string;
+}
+
+export interface DeleteMlInferencePipelineResponse {
+  deleted?: string;
+  updated?: string;
+}
+
+export const deleteMlInferencePipeline = async (
+  args: DeleteMlInferencePipelineApiLogicArgs
+): Promise<DeleteMlInferencePipelineResponse> => {
+  const route = `/internal/enterprise_search/indices/${args.indexName}/ml_inference/pipelines/${args.pipelineName}`;
```

Comment:
Looks like @brianmcgue is renaming this path? https://github.com/elastic/kibana/pull/141015/files#diff-a7d6a7b41796fb8f6a2367de71ddfd881a894eded6629c4ea8e04cf194f98e4cR194

Reply:
Great catch, @byronhulcher! Sorry about that, @TattdCodeMonkey 🙈